### DISCUSSION TITLE
At build time only: Invalid token specified

### DISCUSSION THREAD

**Discussion by n1kalai** [vercel/next.js#21808] [Help]
in development mode, it works nicely, at vercel deployment it deploys but because of working on localhost has problems, but when i try to build and export it has error: "Invalid token specified"

  at Object.<anonymous> (...\node_modules\jwt-decode\build\jwt-decode.cjs.js:1:1147)
 at Module.1TCz (...\.next\server\pages\_app.js:149:68) {
  message: 'Invalid token specified'
}

can anybody tell what i do incorrect?

**Comment by RafaeLS85** [comment]
same error here... any idea?

**Comment by AbdellahEdaoudi** [comment]
The token may be invalid or expired. Make sure you are using a valid token when deploying. The second reason could be "CORS". You should configure it with your deployed project URL like this: 
 app.use((req, res, next) => {
  res.setHeader('Access-Control-Allow-Origin', 'https://linkerfolio.vercel.app');
  res.setHeader('Access-Control-Allow-Methods', 'GET, POST, OPTIONS, PUT, PATCH, DELETE');
  res.setHeader('Access-Control-Allow-Headers', 'Content-Type, Authorization');
  res.setHeader('Access-Control-Allow-Credentials', 'true');
  next();
});
